Renaming packages to io.charlescd (#38)
ZupIT/charlescd: villager/README.md

# CharlesCD Villager

> Villager is a tool that abstracts many cloud providers and provides a unique API to integrate with container registries.

Reviewer: ```suggestion
Villager is a tool that abstracts multiple cloud providers in order to provide a unique API to integrate with container registries.
```

Acredito que trocar o "**and**" para algo de "**causa e efeito**" faz mais sentido. O que acha?

## How Villager works

* Villager has a pluggable architecture that enables the integration with Container Registries of different cloud providers.
* It abstracts every cloud provider API implementations.

> * Provide a unique API to search for tags in your registry, or listen to your registry to check if a given image is being uploaded.

Reviewer: ```suggestion
* Provides a unique API to search for tags, or listen to your registry to check if a given image is being uploaded.
```
